src/main/java/dev/braintrust/eval/Eval.java

```diff
@@ -87,6 +86,9 @@ private void evalOne(String experimentId, DatasetCase<INPUT, OUTPUT> datasetCase
                                 "braintrust.input_json", json(Map.of("input", datasetCase.input())))
                         .setAttribute("braintrust.expected", json(datasetCase.expected()))
                         .startSpan();
+        if (!datasetCase.toOriginMap().isEmpty()) {
```

Comment:
Does checking this here then using it again for JSON rendering make two copies? Add memory pressure?

Reply:
Pushed!